### ISSUE TITLE
Failed to run FFD and python IO models with JModelica 2.4 (and later) in Windows 10 system

### ISSUE THREAD

**Issue by JayHuLBL** [lbl-srg/modelica-buildings#1652]
In Windows 10 platform, models in follow packages can run successfully by JModelica2.1 but fail to run with versions like 2.4 and 2.10.

- `Buildings.ThermalZones.Detailed.Examples.FFD`
- `Buildings.Utilities.IO.Python27.Examples`

for example, with JModelica 2.4,
- running model`Buildings.ThermalZones.Detailed.Examples.FFD.ForcedConvection.mo` gives:
```
C:/JModelica.org-2.10/MinGW/bin/../lib/gcc/x86_64-w64-mingw32/5.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lffd
C:/JModelica.org-2.10/MinGW/bin/../lib/gcc/x86_64-w64-mingw32/5.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lffd
collect2.exe: error: ld returned 1 exit status
mingw32-make[1]: *** [fmume20_] Error 1
mingw32-make: *** [fmume20] Error 2
```
- running model `Buildings.Utilities.IO.Python27.Examples.SimpleRoom.mo` gives:
```
C:\Users\Public\GitFolder\modelica-buildings\Buildings\Resources\Library/win64/ModelicaBuildingsPython2.7.lib: error adding symbols: File in wrong format
collect2.exe: error: ld returned 1 exit status
mingw32-make[1]: *** [fmume20_] Error 1
mingw32-make: *** [fmume20] Error 2
```

**Comment by JayHuLBL** [comment]
Recompiled ffd source code and generated library file for win64 system. Now there are win64 library files for ffd models. For now, running model `Buildings.ThermalZones.Detailed.Examples.FFD.ForcedConvection.mo` gives:
```
C:\Users\Public\GitFolder\modelica-buildings\Buildings\Resources\Library/win64/ffd.lib: error adding symbols: File in wrong format
collect2.exe: error: ld returned 1 exit status
mingw32-make[1]: *** [fmume20_] Error 1
mingw32-make: *** [fmume20] Error 2
```

**Comment by JayHuLBL** [comment]
This issue can be closed as the FFD models can run in Windows OS 64 bit, with both Dymola and Optimica.

The issue related to Python interface classes will be addressed in #2103.
